constants/Firebase.ts

```diff
@@ -0,0 +1,11 @@
+export const firebaseConfig = {
+  apiKey: 'AIzaSyC8Q8S-aeQi5AvvQiojAeeNRk-j4H5nEMk',
```

Comment:
should these be env variables?

Reply:
I don't think we need to hide these variables using env varaibles since they're not secrets. If we need to hide secrets, it should be done in the backend instead.

Reply:
ah ok cool